.azure-pipelines/client.yml

```diff
@@ -1,9 +1,18 @@
 trigger:
 - master
 
-# variables:
+variables:
 # ProjectFile - The project to build and test. This variable is defined in pipeline web ui because we want to be able to provide it at queue time and that isn't supported in yaml yet.
 
+ # public
```

Comment:
Not blocking but I thinking about this a little more we should probably just delete this entire section. Don't worry about doing it for your PR unless you want to as I will clean this as well as the net-lab repo up.

Reply:
I will take care of that as well.